docs/UserGuide.md

```diff
@@ -384,6 +645,33 @@ _Details coming soon ..._
 **Q**: Can I assign multiple roles to a person?<br>
 **A**: No, each person can only have one role at a time. Assigning a new role will replace the existing one.
 
+**Q**: What happens when I delete a wedding?<br>
+**A**: Deleting a wedding will remove all vendor assignments to that wedding and remove the client-wedding relationship. The contacts themselves are not deleted.
+
+**Q**: Can I use the same phone number or email for different contacts?<br>
+**A**: No, phone numbers and email addresses must be unique in the system. You'll receive an error message if you try to add or edit a contact with duplicate information.
+
+**Q**: What happens if I find multiple contacts with the same name?<br>
+**A**: When using name-based commands, if multiple matches are found, the system will show you a list of matching contacts with their indices. You'll need to use the index number to specify which contact you want to work with.
+
+**Q**: How can I see all weddings a vendor is assigned to?<br>
+**A**: Use the `view` command with the vendor's name or index. The system will show all weddings they are assigned to as part of their contact details.
+
+**Q**: Can I search for contacts by partial name match?<br>
+**A**: Yes, use the `find` command which matches partial names. However, note that it matches whole words only (e.g., "John" will match "John Doe" but not "Johnny").
+
+**Q**: What's the difference between `find` and `filter` commands?<br>
+**A**: `find` searches only names and supports partial word matches. `filter` can search across multiple fields (name, role, email, phone, address) but requires exact word matches for names and roles.
```

Comment:
find also allows for multiple name searches. e.g. `find alex david` returns `Alex Yeoh`, `David Li`. which is different from filter. So maybe can include that?